test/FactorialServiceSpec.js

```diff
@@ -15,10 +15,18 @@ describe('FactorialService.js', function() {
     
     it('should null if an invalid param was given', function() {
         expect(FactorialService.calculate(-1)).toBe(null);
-        expect(FactorialService.calculate("1")).toBe(null);
```

Comment:
The controller should be one responsible to parse the input text and send
it to the service.

Look at FactorialController.js

On Tue, Jun 14, 2016, 11:52 PM Guilherme Sanches Pereira <
notifications@github.com> wrote:

> In test/FactorialServiceSpec.js
> <https://github.com/univas/js-factorial/pull/3#discussion_r67092441>:
>
> > @@ -15,10 +15,18 @@ describe('FactorialService.js', function() {
> >
> >      it('should null if an invalid param was given', function() {
> >          expect(FactorialService.calculate(-1)).toBe(null);
> > -        expect(FactorialService.calculate("1")).toBe(null);
>
> Edy, the input used is of type text, then in my opinion should accept
> instead a string equal to "5", for example. Since it can not accept string,
> declaring the type integer would not be more appropriate? thank you for
> attention. :)
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/univas/js-factorial/pull/3/files/6ed811756193b59df50f2c558299a651336358c5#r67092441>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFE-fqUlEVNtyVzP-zCH8_HURKeKcGm1ks5qL2jdgaJpZM4I1l8i>
> .
>